webpack.config.js

```diff
@@ -109,6 +109,13 @@ module.exports = {
             // add errors to webpack instead of warnings
             failOnError: true,
         }),
+
+        new webpack.optimize.CommonsChunkPlugin({
+            name: 'standard',
```

Comment:
I actually tried this and was concerned I was front loading `standard` with modules it does not need. And I didn't fancy creating a separate commons chunk either, for fear of an overall delay. How big is standard after this?

Reply:
standard doesn't change, that just for webpack so it knows the chunks are children of that particular entry point (we only have one entry point).

you can see in the screen shots that effectively a load of common stuff is moved into that `0` chunk

Reply:
from commercial...

Reply:
I thought that if you specify an existing chunk as `name`, then this named chunk becomes the commons chunk, and has the additional commons content appended to it?

Reply:
ah, but then you've set `async`. you should use `async: 'commons'`

Reply:
woah, what's that?

Reply:
![image](https://user-images.githubusercontent.com/867233/35439395-7c1cb952-0292-11e8-998c-56856030bfd4.png)

here's the commons chunk